### ISSUE TITLE
Why classcat?

### ISSUE THREAD

**Issue by zaceno** [kwasniew/hyperapp2-real-world-example#11]
I noticed on the front readme you mention classcat as a dependency. It struck me as curious, since hyperapp 2 has classcat built in to the class prop. So I wanted to check if you were either not aware of that, or if there is a difference between classcat and the built-in behavior that I was not aware of?

**Comment by kwasniew** [comment]
Oh I just wasn't aware. Will fix it soon and another 0.3kB can be shaved off the bundle. Thanks a lot for finding it!

**Comment by kwasniew** [comment]
BTW, do you have any examples how to use the native HA equivalent of classcat?

**Comment by zaceno** [comment]
Sure, the tutorial has a tiny example in this section https://github.com/jorgebucaran/hyperapp/blob/master/docs/src/pages/Tutorial/tutorial.md#composing-the-view-with-reusable-functions-

But basically, anything you would pass to classcat can be passed directly to the class prop.

**Comment by zaceno** [comment]
Btw I don’t blame you in the slightest for not being aware - without any docs, how would you know ?! 😅

**Comment by kwasniew** [comment]
Solved, another 0.1kB shaved off! 
BTW one gotcha is to remove "" when switching from classcat to native HA.
```
class="${{ "nav-link": true, active }}"
class=${{ "nav-link": true, active }}
```
